include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -225,6 +225,16 @@ typedef enum ExecutionMode {
   ORT_PARALLEL = 1,
 } ExecutionMode;
 
+// Set the language projection, default is C, which means it will classify the language not in the list to C also.
+typedef enum OrtLanguageProjection {
+  ORT_PROJECTION_C = 0,  // default
+  ORT_PROJECTION_CPLUSPLUS = 1,
+  ORT_PROJECTION_CSHARP = 2,
+  ORT_PROJECTION_PYTHON = 3,
+  ORT_PROJECTION_JAVA = 4,
+  ORT_PROJECTION_WINML = 5,
+} OrtLanguageProjection;
```

Comment:
We also have nodejs.

Reply:
Yes, also Rust. This list will be expanded in the future.

---
In reply to: [483796987](https://github.com/microsoft/onnxruntime/pull/5023#discussion_r483796987) [](ancestors = 483796987)

Reply:
I meant we already have a nodejs binding in our repository here https://github.com/microsoft/onnxruntime/tree/master/nodejs. So it'll be nice to update that too.